### PR TITLE
[14.0] [MIG] web_widget_model_viewer

### PR DESCRIPTION
**Pull request by TheMule71**
Migration from 12.0 to 14.0

### CODE REVIEW

**Review by andreampiovesana — APPROVED** [review]
go

**Review by dsolanki-initos — APPROVED** [review]
Code review and functional test LGTM



<br class="Apple-interchange-newline">

**Comment by andreampiovesana** [comment]
@simahawk can you merge?

File: web_widget_model_viewer/views/assets.xml [hunk 1]
```diff
@@ -5,15 +5,12 @@
...
+        <xpath expr="//head" position="inside">
             <script
                 type="module"
                 src="/web_widget_model_viewer/static/lib/model-viewer.min.js"
             />
-            <script
-                nomodule="1"
-                src="/web_widget_model_viewer/static/lib/model-viewer-legacy.js"
-            />
+            <!--script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"/-->
```
Comment (eLBati) [inline web_widget_model_viewer/views/assets.xml]: Can this be removed?

Comment (andreampiovesana) [reply web_widget_model_viewer/views/assets.xml]: yes deprecated for new version 1.8

Comment (TheMule71) [reply web_widget_model_viewer/views/assets.xml]: Good spotting, thanks.

**Comment by eLBati** [comment]
Thanks

/ocabot merge nobump

**Comment by simahawk** [comment]
weird, the check on maintainer fails... 
@sbidoul is this a known issue?

**Comment by sbidoul** [comment]
You are hitting https://github.com/OCA/oca-github-bot/issues/122

**Comment by simahawk** [comment]
ahhhh I see. Then sorry @eLBati :)

/ocabot merge nobump

**Pull request merged**